libvisual/CMakeLists.txt

```diff
@@ -185,14 +193,30 @@ ENDIF()
 
 # Git revision
 FIND_PACKAGE(Git)
-CONFIGURE_FILE(
-  "${PROJECT_SOURCE_DIR}/update-version.sh.in"
-  "${PROJECT_BINARY_DIR}/update-version.sh"
-  @ONLY
-)
-ADD_CUSTOM_TARGET(update-version ALL
-  COMMAND /bin/sh ${PROJECT_BINARY_DIR}/update-version.sh
-)
+IF(GIT_FOUND)
```

Comment:
I'm not sure if this check works as intended. This would evaluate true for Git available but no repository below our feet. Git would be run an produce error output then. Is that what we want?

Reply:
I've added `ERROR_QUIET` so that whatever error git shows, it won't be passed to the user and I think this is what we want since if we don't know what revision we are on, we would want to set it to the default

Reply:
The latest version up here does not seem to contain `ERROR_QUIET`. Did you push the latest version?

Reply:
I seemed to have forgot to press enter before I left